beluga_ros/cmake/ament.cmake

```diff
@@ -12,6 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_python REQUIRED)
```

Comment:
Let's have all the `find_package` calls in the same place.

Reply:
Done